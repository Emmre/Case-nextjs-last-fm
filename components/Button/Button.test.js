import { render } from "@testing-library/react"
import "@testing-library/jest-dom"
import Button from "./index"

describe("ButtonComponent", () => {
  it("Should render button component", () => {
    const { getByTestId } = render(<Button data-testid="button" />)
    expect(getByTestId("button")).toBeInTheDocument()
  })

  it("Should match snapshot", () => {
    const { container } = render(<Button data-testid="button" />)
    expect(container).toMatchSnapshot()
  })
})
