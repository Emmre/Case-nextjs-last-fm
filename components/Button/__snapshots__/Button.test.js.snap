// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonComponent Should match snapshot 1`] = `
<div>
  <button
    class="styled__ButtonStyled-sc-6w1m99-0 cBxwXz"
    data-testid="button"
  />
</div>
`;
